.gitpod/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM gitpod/workspace-full
+
+RUN brew install gh && \
+    bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh && sdk install maven 3.8.4 && sdk default maven 3.8.4"
```

Comment:
see also #6491

Reply:
Yep that's the recommended way of handling this, and `mvn` wrapper would be used by default from what I remember